app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
+        self.table_name = "actors"
+

```

Comment:
Before insert data in db we need to make sure table is existing  or we need to create it